services/server/src/handlers/contactRequest.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from "express";
+import zod from "zod";
+import { string, email } from "@/validation/utils";
+import safeRequest from "express-async-handler";
+import { IContactRequest } from "@litespace/types";
+import { contactRequests } from "@litespace/models";
+import { Telegram } from "@litespace/radio";
+import { telegramConfig } from "@/constants";
+
+const addPayload = zod.object({
+  name: string,
+  email: email,
+  title: string,
+  message: string,
+});
+
+async function add(req: Request, res: Response, _: NextFunction) {
+  const payload: IContactRequest.CreatePayload = addPayload.parse(req.body);
+  await contactRequests.create([payload]);
+  res.status(200);
+
+  const telegram = new Telegram(telegramConfig.token);
```

Comment:
We should initialize it once and reuse it on-demand. Simliar to what we did [here](https://github.com/litespace-org/litespace/blob/ee2b00f63ec7bcd589b6c7eaf62505cda1d0b8d0/services/apollo/src/telegram.ts#L4). We can put the instance into a lib file `lib/telegram.ts`.